FirebaseAdmin/FirebaseAdmin.Tests/Auth/IdTokenVerificationTest.cs

```diff
@@ -0,0 +1,466 @@
+// Copyright 2018, Google Inc. All rights reserved.
```

Comment:
Is this a new file? Should this be 2020?

Reply:
Done